src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/Serialization/RazorDiagnosticJsonConverter.cs

```diff
@@ -27,15 +27,16 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             }
 
             var diagnostic = JObject.Load(reader);
+            var id = diagnostic[nameof(RazorDiagnostic.Id)].Value<string>();
+            var severity = diagnostic[nameof(RazorDiagnostic.Severity)].Value<int>();
+            var message = diagnostic[RazorDiagnosticMessageKey].Value<string>();
+
             var span = diagnostic[nameof(RazorDiagnostic.Span)].Value<JObject>();
+            var filePath = span[nameof(SourceSpan.FilePath)].Value<string>();
```

Comment:
Reorganized these to match the order used while writing. Makes it easier to inspect the code visually.